cairo/ethereum/cancun/trie.cairo

```diff
@@ -963,7 +949,7 @@ func _get_branches{poseidon_ptr: PoseidonBuiltin*}(obj: MappingBytesBytes, level
         assert value = value_15;
         assert value_set = 1;
     }
-    %{ ids.value_set = memory.get(fp + 2) or 0 %}
+    %{ value_set_or_zero %}
```

Comment:
I'd advocate for a more detailed naming like `fp_plus_2_or_0`. We also probably would like to have a generic hint. Thinking that since it's basically just a string, we could encode more data in it to have cairo 1 like hint (with src and dst)

Reply:
yes, I was thinking of generic hint like this that would take a dynamic argument from the hint name. 

e.g. `fp_offset_or_0(offset: int)` and then use this as function param 🤔